scripts/prerequisites.sh

```diff
@@ -133,9 +133,10 @@ essential="\
     wget \
 "
 
-# add python3-distutils only for 18.04 (and later should be added for newer)
+# add python3-distutils only for 18.04 and later
 # because it doesn't exist as separate package in older Ubuntu releases
-[ `/usr/bin/lsb_release -s -r` = "18.04" ] && essential="${essential} python3-distutils"
+release_numerical=$(echo "$release | tr -d .)
+[ -n "$release" ] && [ "$release_numerical" -ge "1804" ] &&  essential="${essential} python3-distutils"
```

Comment:
Comment from myself:  It might look nicer to use a real if-statement here, but I just continued with the style that was already used here.  (I also write expressions like this in my own scripts because it is efficient.)

Reply:
Sorry, the missing quote.  Bugfixed and force-pushed now.